packages/circuit-ui/components/ToastContext/ToastContext.tsx

```diff
@@ -50,10 +51,14 @@ export interface ToastProviderProps {
    * The ToastProvider should wrap your entire application.
    */
   children: ReactNode;
+  position?: 'bottom' | 'top' | 'top-right';
```

Comment:
Please add a doc comment for the prop to describe its purpose and encourage users to stick with the default if possible.

Reply:
added in [1465d68](https://github.com/sumup-oss/circuit-ui/pull/2577/commits/1465d686c9b47049095c94bad6d9e2fe520b82f4) and in [c1bfb2c](https://github.com/sumup-oss/circuit-ui/pull/2577/commits/c1bfb2c48137a968dc3595171b9827eeb78f8454), do you think this will be enough?

Reply:
Yes, this is great 👏🏻